### PR TITLE
 Disney World

### PR DESCRIPTION
**Pull request by ibr5500**
## In this [PR] I have:

> - Created React App.
> - Installed [React Redux](https://react-redux.js.org/), [Redux Logger](https://www.npmjs.com/package/redux-logger) and [React Router](https://reactrouter.com/web/guides/quick-start).
> - Created routes and view components (Characters, and Details).
> - Created directories for Redux state slice files (characters).
> - Fetched data from the [Disney API](https://disney-world.netlify.app/).
> - Stored the selected data in the Redux store.
> - Rendered the details onClick on a character using useSelector() Redux Hook.
> - Ensure that each Character has unique route.
> - Deployed the project.

### CODE REVIEW

File: src/components/Details.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+import { useParams } from 'react-router-dom';
+
+const Details = () => {
+  const character = useParams();
+  const details = useSelector((state) => state.characters);
+  const charDetails = details.filter((char) => +char.id === +character.id);
+
+  return (
+    <div className="details-container">
+      {charDetails.map((char, index) => (
```
Comment (zieeco) [inline src/components/Details.js:12]: **[REQUIRED CHANGES]**

- Your app is working fine but a user can't actually search for a particular Disney as this is part of the project requirement. There's no input or search box in your design where users can search. Please, kinly add this missing feature which is key in your app functionality. see the image below:


SEARCH

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @ibr5500,


Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice design. :fire: 
- Descriptive PR. :100: 
- App working as expected. :fire: 
- Good code organization. ✔️



## Required Changes ♻️
_Check the comments under the review._





## Optional suggestions

**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Thank you @zieeco for the code review.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,99 @@
...
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+### Please give a ⭐️ if you like this project!
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (Lordkaito) [inline README.md:99]: - [ ] Please, as the previous reviewer mentioned, please make sure to add the Acknowledgement section to your readme and give credit to Nelson Sakwa as the author of the original design.

**Review by Lordkaito — CHANGES_REQUESTED** [review]
Hi @ibr5500, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by ibr5500** [comment]
Thank you @Lordkaito for the code review.

**Review by Temmarie — APPROVED** [review]
# Status: Approved ✔️ 
## Hi @ibr5500,
 
### You have done extremely well in implementing the project requirements and the previously required changes. Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### Congratulations! 🎉 

#### Cheers and Happy coding!👏👏👏

#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
------
_As described in the [Code reviews limits policy](<https://microverse.zendesk.com/hc/en-us/articles/1500004088561>) you have a limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](<https://airtable.com/shrQAqnBwek5a0O0s>)._

**Comment by ibr5500** [comment]
Thank you @Temmarie for the code review.

**Pull request merged**